advanced-integration-dotnet/README.md

```diff
@@ -0,0 +1,20 @@
+# Advanced Integration Example (.NET)
```

Comment:
Sounds good. So if we have dontet with react, it will be something like `dotnet-react`?

Reply:
Also, can we remove the `advanced-integration/README.md` file? it is not relevant anymore. Since we will have `README.md` file inside each language folder.